detection-rules/body_advance_fee_freemail_sender.yml

```
@@ -0,0 +1,31 @@
+name: "Body: Advance Fee attempt from free  first-time sender"
```

```suggestion
name: "Advance Fee attempt from first-time sender"
```
doesn't look like this is limited to freemail senders. also, there's no FTS snippet in this rule

---

didn't take a deep look at this rule but noticed these 2 things from a quick glance